Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -289,7 +289,7 @@ open class ComposerVC: _ViewController,
 
     open var cooldownTracker: CooldownTracker = CooldownTracker(timer: ScheduledStreamTimer(interval: 1))
 
-    public var enrichUrlDebouncer = Debouncer(0.3, queue: .main)
+    public var enrichUrlDebouncer = Debouncer(0.4, queue: .main)
```

Comment:
Is there any visible delay because of this change?

Reply:
There was before the actual fix. But it is better to put 0.4 since it is actually the value proposed in the Figma designs. I initially put it shorter because it was better, but decided to put the actual value from Figma